flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/DataGenerators.java

```diff
@@ -38,11 +38,11 @@
 import org.apache.avro.SchemaBuilder;
 import org.apache.avro.generic.GenericData;
 import org.apache.avro.util.Utf8;
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
Nit: we use `UnsupportedOperationException` in other places, should we keep consistency?